app/models/user.rb

```diff
@@ -1,5 +1,12 @@
 class User < ApplicationRecord
+  has_many :owned_accounts, class_name: 'Account', foreign_key: 'owner_id'
+  has_many :owned_account_debits, class_name: 'AccountDebit', foreign_key: 'owner_id'
+  has_many :owned_account_credit_cards, class_name: 'AccountCreditCard', foreign_key: 'owner_id'
+  has_many :owned_account_objectives, class_name: 'AccountObjective', foreign_key: 'owner_id'
```

Comment:
Here we can remove this relations because we can have the same through `owned_groups`.

```suggestion
```